README.md

```diff
@@ -42,12 +42,25 @@ New developers may find the notes in [CONTRIBUTING](https://github.com/JuliaLang
 
 ## Currently Supported Platforms
 
-- **Linux**
-- **MacOS**
-- **FreeBSD**
-- **Windows**
-
-All systems are supported with both x86/64 (64-bit) and x86 (32-bit, except MacOS) architectures. Support for [ARM](https://github.com/JuliaLang/julia/blob/master/README.arm.md), AARCH64, and POWER8 (little-endian) has been added recently.
+Julia is built and tested regularly on the following platforms:
+
+| Operating System | Architecture     | CI | Binaries |
+|:----------------:|:----------------:|:--:|:--------:|
+| Linux            | x86-64 (64-bit)  | ✓  | ✓        |
+|                  | i686 (32-bit)    | ✓  | ✓        |
+|                  | PowerPC (64-bit) |    | ✓        |
+|                  | ARM v7 (32-bit)  |    | ✓        |
+|                  | AArch64 (64-bit) |    |          |
+| macOS 10.8+      | x86-64 (64-bit)  | ✓  | ✓        |
+| Windows 7+       | x86-64 (64-bit)  | ✓  | ✓        |
+|                  | i686 (32-bit)    | ✓  | ✓        |
+| FreeBSD 11.0+    | x86-64 (64-bit)  |    |          |
```

Comment:
Right. I'm not sure what you mean by "CI is waiting on it to stay green long enough." I believe that builtbot uses system libraries, which will bypass the stuff I did to get it building from scratch.